lib/stringify.js

```diff
@@ -56,7 +56,7 @@ var isNonNullishPrimitive = function isNonNullishPrimitive(v) {
         || typeof v === 'bigint';
 };
 
-var scSym = Symbol('SideChannel');
+var scSym = Object.create(null);
```

Comment:
Unfortunately Object.create also can’t be relied upon.
```suggestion
var sentinel = {};
```

Reply:
> 

Submitted new code. for this question, i didn't understand why it was unreliable. two objects created by `Object.create`, return value is false when judged by ===.

Reply:
I see, polyfill is provided in MDN and it doesn't support null as the first parameter, is that the reason?

Reply:
It's impossible to polyfill null objects.

We support down to IE 6, which doesn't have Object.create or `__proto__`.